specification/cognitiveservices/data-plane/FormRecognizer/readme.md

```diff
@@ -0,0 +1,71 @@
+# Cognitive Services Form Recognizer SDK
+
+> see https://aka.ms/autorest
+
+Configuration for generating Form Recognizer SDK.
+
+The current release is `release_1_0`.
+
+``` yaml
+
+tag: release_1_0
+add-credentials: true
+openapi-type: data-plane
+```
+# Releases
+
+### Release 1.0
+These settings apply only when `--tag=release_1_0` is specified on the command line.
+
+``` yaml $(tag) == 'release_1_0'
+input-file: preview/v1.0/formrecognizer.json
+```
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-go
+  - repo: azure-sdk-for-node
+  - repo: azure-sdk-for-ruby
+    after_scripts:
+      - bundle install && rake arm:regen_all_profiles['azure_cognitiveservices_formrecognizer']
+```
+
+
+## CSharp Settings
+These settings apply only when `--csharp` is specified on the command line.
+``` yaml $(csharp)
+csharp:
+  sync-methods: None
+  license-header: MICROSOFT_MIT_NO_VERSION
+  azure-arm: false
+  namespace: Microsoft.Azure.CognitiveServices.Vision.FormRecognizer
+  output-folder: $(csharp-sdks-folder)/CognitiveServices/dataPlane/Vision/FormRecognizer/Generated
+  clear-output-folder: true
+```
+
+## Go
+
+See configuration in [readme.go.md](./readme.go.md)
+
+## Java
+
+These settings apply only when `--java` is specified on the command line.
+Please also specify `--azure-libraries-for-java-folder=<path to the root directory of your azure-libraries-for-java clone>`.
+
+``` yaml $(java)
+java:
+  azure-arm: true
+  namespace: com.microsoft.azure.cognitiveservices.vision.formrecognizer
+  license-header: MICROSOFT_MIT_NO_CODEGEN
+  payload-flattening-threshold: 1
+  output-folder: $(azure-libraries-for-java-folder)/cognitiveservices/data-plane/vision/formrecognizer
```

Comment:
`$(azure-libraries-for-java-folder)/cognitiveservices/data-plane/formrecognizer`

Reply:
as per sdk PR:
https://github.com/Azure/azure-sdk-pr/issues/8

Reply:
done.